glide.yaml

```diff
@@ -1,9 +1,11 @@
 package: github.com/giantswarm/cert-operator
 import:
 - package: github.com/golang/protobuf
-  version: 8ee79997227bf9b34611aee7946ae64735e6fd93
+  version: 2bba0603135d7d7f5cb73b2125beeda19c09f4ef
+- package: google.golang.org/grpc
```

Comment:
This is one of those dumb vanity go library things, and seemed to screw glide up